### PR TITLE
Slider

### PR DESCRIPTION
**Pull request by tomgatzgates**
- [x] Min, max and step
- [ ] Show value

### CODE REVIEW

File: components/Slider/Slider.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  border-radius: 1em;
+  cursor: pointer;
+  width: 100%;
+}
+
+.root:focus {
+  outline: none;
+}
+
+.root::-webkit-slider-thumb {
```
Comment (iest) [inline components/Slider/Slider.css]: What does this look like on Firefox or IE?

We're using autoprefixer - so if these rules are based on a standard we should be able to do without the vendor prefixes.

**Comment by tomgatzgates** [comment]
Havent tested. I was hoping auto prefix would pick up the root (slider bg). I need to check the slider thumb though. 

Just playing round and trying to learn at the moment so will add compatibility when I get the functionality working :)

**Comment by iest** [comment]
Awesome!

You need to get eslinter working on your sublime :)

**Comment by iest** [comment]
Closing as we implemented elsewhere.

**Pull request closed without merging**